ocaml/src/fragment_cache.ml

```diff
@@ -438,24 +441,33 @@ class blob_cache root ~(max_size:int64) ~rocksdb_max_open_files
          Lwt.return None
         )
 
-    method lookup bid oid =
-      self # _lookup bid oid read_it >>= function
-      | Some data ->
-         Lwt.return (Some (data, []))
-      | None ->
-         Lwt.return_none
-
-    method lookup2 bid oid slices =
-      self # _lookup
-           bid oid
-           (fun fd ~len ->
-            Lwt_list.iter_s
-              (fun (offset, length, target, offset') ->
-               Lwt_unix.lseek fd offset Lwt_unix.SEEK_SET >>= fun _ ->
-               Lwt_extra2.read_all_lwt_bytes_exact fd target offset' length)
-              slices) >>= function
-      | Some () -> Lwt.return (true, [])
-      | None -> Lwt.return (false, [])
+    method lookup ?(timeout= 10.0) bid oid =
```

Comment:
Does having a hard coded value make sense here? Maybe it's better to force the caller to supply a reasonable value.
Fwiw there's another hard coded value in fragment_cache_alba.ml.